.github/workflows/main.yml

```diff
@@ -5,33 +5,43 @@ on:
     branches:
       - main
 
+env:
+  CARGO_INCREMENTAL: 0
+  CARGO_PROFILE_DEV_DEBUG: 0
+
 jobs:
   benchmark:
     runs-on: ubuntu-latest
     steps:
       - name: Check out repository
         uses: actions/checkout@v2
 
-      - name: Set up go
-        uses: actions/setup-go@v2
+      - name: Set up Go
+        uses: actions/setup-go@v4 # v4 uses caching of Go out of the box
         with:
           go-version: '1.20'
-
       - name: Install nats-server
-        run: go install github.com/nats-io/nats-server/v2@latest
+        run: go install github.com/nats-io/nats-server/v2@main
```

Comment:
1/5 we should probably use the latest server _release_ rather than `main` for benchmarking

Reply:
`curl -s https://api.github.com/repos/nats-io/nats-server/releases/latest | jq -r '.tag_name'`

Reply:
After I added the caching here, this would require getting the current version of the server to use it in the cache key (for now I only build the key from the os and the go version)